app/assets/javascripts/select-all.js

```diff
@@ -0,0 +1,23 @@
+(function (Modules) {
+  "use strict";
+
+  Modules.SelectAll = function () {
+    var that = this;
+
+    that.start = function (element) {
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/290/commits/d4fd166a4ef59282c85bf2d178a0748e0c77dbf3#diff-1bf7091d9f272b27a4976caf9081ac8cR5